src/lib.rs

```diff
@@ -2218,6 +2254,16 @@ impl<'a> Face<'a> {
         Some(())
     }
 
+    /// Sets the foreground color of the face. Is used by the COLRv1 table.
+    pub fn set_foreground_color(&mut self, color: RgbaColor) {
```

Comment:
Hm... I would prefer passing it with each colr1 "outline" call. Would it be possible?

Reply:
You meab `colr::paint`? I think it should be doable, the only reason I changed it is because I thought it might be nice to align it's behaviour to how we deal with variable fonts.

Reply:
I added it to `paint_colr_glyph` instead.